settings/supervisor/database-migrate.conf

```
@@ -0,0 +1,13 @@
+[program:movay-database-migrate]
+command=/usr/local/bin/php /app/bin/console.php database:migration:migrate
+numprocs=1
+user=application
+autostart=true
+autorestart=unexpected
+exitcodes=0
```

```suggestion
autorestart=unexpected
exitcodes=0
```

retry if the expected status code was not returned (maybe same initial connection issues?)